### PR TITLE
Remove plumbing for ignored --workers option

### PR DESCRIPTION
**Pull request by eseidelGoogle**
I've left the option in, but removed all the dead plumbing.
The workers parameter is ignored recent versions of
package:coverage.

### CODE REVIEW

File: lib/src/collect_lcov.dart [hunk 1]
```diff
@@ -62,21 +62,20 @@ class LcovCollector {
...
-  Future<String> convertVmReportsToLcov(Directory directoryContainingVmReports,
-      {int workers: 1}) async {
+  Future<String> convertVmReportsToLcov(
+      Directory directoryContainingVmReports) async {
     var reportFiles = await directoryContainingVmReports
         .list(recursive: false, followLinks: false)
         .toList();
 
-    var hitmap = await parseCoverage(reportFiles as Iterable<File>, workers);
+    var hitmap = await parseCoverage(reportFiles as Iterable<File>, null);
```
Comment (eseidelGoogle) [inline lib/src/collect_lcov.dart]: All this plumbing was just for this call, however if you look at the parseCoverage implementation, this parameter is ignored:
https://github.com/dart-lang/coverage/blob/master/lib/src/hitmap.dart#L69

**Comment by nilsdoehring** [comment]
Thanks @eseidelGoogle! By the way, I'm currently not able to push to pub thanks to [this gem](https://github.com/dart-lang/pub/issues/1442), working on it.

**Comment by eseidelGoogle** [comment]
If you add me as an uploader, I can attempt to publish.  I may also just look at fixing that pub bug myself if I can reproduce it.

**Comment by nilsdoehring** [comment]
Added you as an uploader. Also gave you write access to this repo.
I actually found and posted a resolution for that pub bug, let's hope it gets accepted – or someone moves forward with verifying the pure Dart solution mentioned (probably referring to [archive](https://github.com/brendan-duncan/archive) package).

**Comment by nilsdoehring** [comment]
@eseidelGoogle *bump* - it doesn't look like the issue will get fixed, soon... could you try and publish? Thanks!

**Comment by eseidelGoogle** [comment]
Yup. Hope to try tonight.  (~10 hours from now.)

**Comment by eseidelGoogle** [comment]
@nilsdoehring it seems like before we push a new version we need to update the constraints in the pubspec, no?  1.22 is the new minimum SDK right (not 1.9 as it is now), similarly the constraint on coverage?

**Comment by eseidelGoogle** [comment]
Moved discussion to https://github.com/block-forest/dart-coveralls/issues/65.

**Comment by eseidelGoogle** [comment]
I've pushed 0.5.0: https://pub.dartlang.org/packages/dart_coveralls

**Pull request merged**